.mdl_style.rb

```diff
@@ -3,6 +3,8 @@
 rule 'MD007', :indent => 2
 # Ignore line length in code blocks and tables
 rule 'MD013', :ignore_code_blocks => true, :tables => false
+# Question marks are okay for us
+rule 'MD026', :punctuation => ".,;:!"
```

Comment:
https://pymarkdown.readthedocs.io/en/stable/plugins/rule_md026/ states that question marks are OK by default. It looks like the default list is `.,;:!。，；：！` so you've removed some characters? I still don't understand what this is supposed to do.

Reply:
The "What is LinuxBoot?" is the last remaining one we have. We could rephrase it.

The [other markdown linter](#183) allows for questions by default because of FAQ style documents:
<https://github.com/DavidAnson/markdownlint/blob/main/doc/md026.md>

I find that reasonable.

While we do have an FAQ chapter, the only question in there for now is just written in bold text though.